proposals.md

```diff
@@ -0,0 +1,18 @@
+# Proposals
+This directory contains the schedule of communication seminar and design proposals.
+
+## Communicaiton Seminar
+A communication seminar will be organized to discuss the details after a proposal is reviewed. Community members are welcomed to attend.
+
+### Seminar Schedule
+The schedule contains all ongoing proposals, please find the information below and join the seminar if you are interested. 
```

Comment:
```suggestion
The schedule contains all ongoing proposals, and please find the information below and join the seminar if you are interested. 
```

Reply:
Note: Please do not combine two sentences with a comma. ^_^ Instead, you can use a conjunction (e.g. "and") or just separate them with a period.